internal/monitors/logstash/logstash/metadata.yaml

```diff
@@ -0,0 +1,427 @@
+      description: Queue size in pipelines
+      default: false
+      type: gauge
+      group: pipeline
+    node.stats.pipelines.reloads.failures:
+      description: Number of failed reloads
+      default: false
+      type: cumulative
+      group: pipeline
+    node.stats.pipelines.reloads.successes:
+      description: Number of successful reloads
+      default: false
+      type: cumulative
+      group: pipeline
+    node.stats.events.events.duration_in_millis:
+      description: Duration of events
+      default: true
+      type: gauge
+      group: events
+    node.stats.events.events.filtered:
+      description: Number of filtered events
+      default: true
+      type: cumulative
+      group: events
+    node.stats.events.events.in:
+      description: Number of incoming events
+      default: true
+      type: cumulative
+      group: events
+    node.stats.events.events.out:
+      description: Number of outgoing events
+      default: true
+      type: cumulative
+      group: events
+    node.stats.events.events.queue_push_duration_in_millis:
+      description: Queue push duration of events
+      default: false
+      type: gauge
+      group: events
+    node.plugins.total:
+      description: Number of plugins
+      default: true
+      type: cumulative
+      group: plugins
+    node.stats.process.process.cpu.load_average.1m:
+      description: CPU Load average in 1 minute
+      default: false
+      type: gauge
+      group: process
+    node.stats.process.process.cpu.load_average.5m:
+      description: CPU Load average in 5 minutes
+      default: false
+      type: gauge
+      group: process
+    node.stats.process.process.cpu.load_average.15m:
+      description: CPU Load average in 15 minutes
+      default: false
+      type: gauge
+      group: process
+    node.stats.process.process.cpu.percent:
+      description: CPU usage in percent
+      default: true
+      type: gauge
+      group: process
+    node.stats.process.process.cpu.total_in_millis:
+      description: Total CPU time (in milliseconds) used by the process on which the process is running
+      default: false
+      type: gauge
+      group: process
+    node.stats.process.process.max_file_descriptors:
+      description: Number of opened file descriptors associated with the current process
+      default: false
+      type: cumulative
+      group: process
+    node.stats.process.process.mem.total_virtual_in_bytes:
+      description: Size of the virtual memory of this process
+      default: false
+      type: gauge
+      group: process
+    node.stats.process.process.open_file_descriptors:
+      description: Number of currently open file descriptors
+      default: false
+      type: cumulative
+      group: process
+    node.stats.process.process.peak_open_file_descriptors:
+      description: Peak number of currently open file descriptors
+      default: false
+      type: cumulative
```

Comment:
Same thing.  Being a "peak" value implies it might be monotonically increasing but it still isn't really the semantics of a cumulative counter.